File: py2cpp/examples/example4.cpp
#include <fmt/core.h>
#include <cstdlib> // import atof
#include <utility> // import swap

int main(int argc, const char* argv[])
{
    auto a = std::atof(argv[1]);
    auto b = std::atof(argv[2]);
    std::swap(a, b);
    fmt::print("The answer is {}.\n", a - b);
}

File: py2cpp/examples/example4.py
#! /usr/bin/env python3
import sys

def main(args):
    a = float(args[1])
    b = float(args[2])
    a, b = b, a  # swap
    print('The answer is {}.'.format(a - b))

if __name__ == '__main__':
    main(sys.argv)
